lib/widgets/content.dart

```diff
@@ -1057,9 +1060,19 @@ class AvatarImage extends StatelessWidget {
       null          => null, // TODO(#255): handle computing gravatars
       var avatarUrl => store.tryResolveUrl(avatarUrl),
     };
-    return (resolvedUrl == null)
-      ? const SizedBox.shrink()
-      : RealmContentNetworkImage(resolvedUrl, filterQuality: FilterQuality.medium, fit: BoxFit.cover);
+
+    if (resolvedUrl == null) {
+      return const SizedBox.shrink();
+    }
+
+    final avatarUrl = AvatarUrl.fromUserData(resolvedUrl: resolvedUrl);
+    final physicalSize = ((MediaQuery.maybeDevicePixelRatioOf(context) ?? 1) * size).ceil();
```

Comment:
Hmm, interesting. Where does 1 come from as a default?

I think if we really had to use a default here, it'd be better to be more toward the high end of likely values. That'd mean we get an image that has enough pixels it shouldn't look blurry.

But I don't know a reason why we should need a default for this — there should always be a `MediaQuery` ancestor. And the previous revision didn't have a default. So instead of `?? 1`, that would be spelled
```dart
MediaQuery.maybeDevicePixelRatioOf(context)!
```
or more concisely
```dart
MediaQuery.devicePixelRatioOf(context)
```
.